### ISSUE TITLE
Formating Symbols get removed when sending over Telegram

### ISSUE THREAD

**Issue by reckel-jm** [42wim/matterbridge#1748]
**Describe the bug**
If I send a formatted message over Telegram, all the formatting symbols such as `**` or `__` get removed from the Debug and from the receiving channels.

**To Reproduce**
Send any formatted message in Telegram.

**Expected behavior**
The formatting symbols should be transfered correctly or converted into the format of the recipient.  

**Environment (please complete the following information):**
 - OS: Linux
 - Matterbridge version: 1.24.0 65c7ac80
 - not self compiled

**Additional context**
```
[telegram.mytelegram]
Token=...
RemoteNickFormat="__Nachricht von__ **{NICK}** __über {PROTOCOL}__: \n \n"
MessageFormat="Markdown"
IgnoreMessages = "#local"
```

**Comment by reckel-jm** [comment]
My pull request #1749 should fix that problem!

**Comment by 42wim** [comment]
Thanks, closed by #1749
